fbw-a32nx/docs/a320-simvars.md

```diff
@@ -2697,21 +2676,19 @@ In the variables below, {number} should be replaced with one item in the set: {
 - A32NX_PNEU_ENG_{number}_INTERMEDIATE_TRANSDUCER_PRESSURE:
     - Pressure measured at the intermediate pressure transducer, -1 if no output
     - psi
+    - Only on the A380X
```

Comment:
If this is specific to the A380, would it make sense to record it in a380-simvars.md?

Reply:
> If this is specific to the A380, would it make sense to record it in a380-simvars.md?

That would make a lot of sense! What's the policy on where we record the vars? Do we only put A380-exclusive variables in that file or do we also duplicate the common ones in that file?

Reply:
The way I did it in #7945 is to have all shared vars in the a320 doc and the a380 specific vars in the a380 doc. But I see that it also makes sense to also have all bleed vars in a single document to not have to jump between the two. I'll leave it up to you as we don't seem to have a "policy" as such 😄.